qualtran/_infra/bloq_test.py

```diff
@@ -20,6 +20,7 @@
 from attrs import frozen
 
 from qualtran import Bloq, CompositeBloq, Side, Signature
+from qualtran._infra.gate_with_registers import get_named_qubits
```

Comment:
yes, that's the case right now. If users are constructing a Cirq circuit like we do in tests, then it can be useful to obtain a bunch of named qubits from the signature of the bloq. We are currently using this method in notebooks and tests only.